lmfdb/utils/search_parsing.py

```diff
@@ -339,7 +340,7 @@ def parse_rational(inp, query, qfield):
 
 @search_parser(clean_info=True, prep_ranges=True) # see SearchParser.__call__ for actual arguments when calling
 def parse_ints(inp, query, qfield, parse_singleton=int):
-    if LIST_RE.match(inp):
+    if SIGNED_LIST_RE.match(inp):
```

Comment:
You're changing the behavior of `parse_ints` everywhere in the LMFDB.  Now, I agree that the name of `parse_ints` is not ideal (we should perhaps change it to `parse_positive_ints`, and also change the name of the regular expression from `LIST_RE` to `POSITIVE_INTS_RE`.  But you've removed some error checking that was useful.

Besides, you're no longer using the parser in your application.  I would probably just undo your changes to this file and leave it for another day, unless you'd like to change various names to make it clearer what is being done.  That's probably better for another PR anyway.

Reply:
I still use `parse_ints` in the ecnf version unless I'm mistaken...

Reply:
In any case, I'm happy to revert my changes and make a separate issue.

Reply:
Okay, I've reverted my changes to `parse_ints`. The new issue is here: https://github.com/LMFDB/lmfdb/issues/3825